include/oneapi/dpl/pstl/execution_defs.h

```diff
@@ -82,6 +82,8 @@ inline constexpr bool is_execution_policy_v = oneapi::dpl::execution::is_executi
 } // namespace v1
 } // namespace execution
 
+using oneapi::dpl::execution::is_execution_policy_v;
```

Comment:
This adds what is proposed in https://github.com/uxlfoundation/oneAPI-spec/issues/558, but only for `is_execution_policy_v`, not `is_execution_policy`.  
I'm not sure if this has been discussed already in this PR, but we should probably pair this with a corresponding spec change, and include both here if were going to do it.

Reply:
It probably makes sense to break off this change as a separate PR and include both `is_execution_policy` and `is_execution_policy_v` together to satisfy https://github.com/uxlfoundation/oneAPI-spec/pull/567.

Reply:
> together to satisfy [uxlfoundation/oneAPI-spec#567](https://github.com/uxlfoundation/oneAPI-spec/pull/567).
The PR (Part1 and Part2) contain more than 20 lines `...requires oneapi::dpl::is_execution_policy_v...`.
So, I would suggest keeping the injection `using oneapi::dpl::execution::is_execution_policy_v;` fpr now.
And after  - create a separate PR that implements the spec.

Reply:
Tagging https://github.com/oneapi-src/oneDPL/pull/1457#discussion_r1683407090, as the previous discussion.

Reply:
I remember that I asked Alexey about that. And he told me that we should fix the oneDPL spec first.
Regarding the current PR h told that you can use an alias for is_execution_policy_v.

Reply:
If we want to wait for the specification changes to be official before making changes in implementation, then in this PR we should move back to explicitly using `oneapi::dpl::execution::is_execution_policy[_v]` in this PR, and remove this alias.    

We are spending far more time discussing it then it takes to just make this change with a query replace.

Reply:
I've created #1803 and #1804, feel free to review and apply them if they are useful to you.

Reply:
I discussed that trait with Ruslan yesterday. He told me that the spec tells "the both traits (oneapi::dpl::is_execution_policy and oneapi::dpl::is_execution_policy_v) should be available as in oneapi::dpl and in oneapi::dpl::exeution as well".
So, we came to agreement just making visibility of both traits, it is enough change for now.

Reply:
Sure, either option works for me.  As long as they both go together, I think it is good, and implements the specification proposal as written.